<launch>

    <!-- <include file="$(find pointcloud_utils)/launch/shell_lidar_calibration.launch" /> -->

	<node pkg="pointcloud_utils" type="shell_pointcloud_grid_parser" name="pc_grid_parser" required="true" output="screen">
		<param name="cloud_topic" 		value="/airsim_node/PhysXCar/lidar/VLP16_1" />
		<!-- <param name="base_frame"      value="PhysXCar/odom_local_ned" /> -->
		<param name="base_frame"      value="VLP16_1" />
		<param name="lidar_frame"     value="VLP16_1" />
		<param name="image_pub_topic" 	value="/grid_image" />
		<param name="costmap_pub_topic"	value="/map" />

		<param name="constant_map_resolution" value="true" />
		<param name="constant_map_size" value="true" />
		<param name="use_bounds" 		value="true" />
		<param name="recalculate_dynamic_bounds" value="false" />
		<param name="binary_map" value="true" />

		<param name="map_resolution" 	value="0.1" /> #m/cell
		<param name="map_width" 		value="512" /> #cell was 1024
		<param name="map_height" 		value="512" /> #cell was 1024

		<param name="z_scale_max" 		value="0.2" /> #[m] #was 5
		<param name="z_scale_min" 		value="-0.5" /> #[m] #was 0

		<param name="centered_x"      value="false" />
		<param name="centered_y"      value="true" />

		#bounds        
		<param name="x_min" value="-10"  /> #was -10
		<param name="x_max" value="200"  />
		<param name="y_min" value="-20" />
		<param name="y_max" value="20"  />
		<param name="z_min" value="0" />
		<param name="z_max" value="100"  />
	</node> -->

	
</launch>